components/automate-ui/src/app/components/resource-dropdown/resource-dropdown.component.html

```diff
@@ -0,0 +1,42 @@
+<label>{{ objectNounPlural | capitalize }}</label>
+<div class="dropdown-wrap">
+  <button class="dropdown-button" aria-label="Select resources"
```

Comment:
will that aria-label change to the correct resource name?

Reply:
Fixed